config/jobs/kubernetes/sig-testing/periodic-kubernetes-compatibility-version-e2e.yaml

```diff
@@ -0,0 +1,51 @@
+periodics:
+- interval: 24h
+  cluster: k8s-infra-prow-build
+  name: pull-kubernetes-e2e-kind-emulated-version
```

Comment:
name it ci-kubernetes-e2e-* for periodic jobs, pull-kubernetes-e2e are for presubmits in general

Reply:
Done, renamed to:
```
name: ci-kubernetes-e2e-kind-compatibility-versions
```